template/script.sh.erb

```diff
@@ -51,7 +53,10 @@ code-server \
     --auth="password" \
     --bind-addr="0.0.0.0:${port}" \
     --disable-telemetry \
+    <% if r_version != '4.5' %>
+    # code-server 4.5 will not accept this argument
     --extra-extensions-dir="$CODE_SERVER_DATAROOT/extensions" \
```

Comment:
I was not able to get the `--extra-extensions-dir` to function correctly with `4.5.1`. Any suggestions to fix welcome.

Reply:
`r_version`?  I think you've got R on the mind. `<=` is likely safer though.

Reply:
:man_facepalming: Yes, yes I do. I will change that variable name and the equality check.